app/components/op_primer/email_updates_mode_selector_component.rb

```diff
@@ -28,48 +28,48 @@
 # See COPYRIGHT and LICENSE files for more details.
 #++
 
-module Meetings
-  class SidePanel::NotificationsButtonComponent < ApplicationComponent
-    include ApplicationHelper
+module OpPrimer
+  class EmailUpdatesModeSelectorComponent < Primer::Component
     include OpTurbo::Streamable
     include OpPrimer::ComponentHelpers
 
-    def initialize(meeting:)
+    def initialize(toggle:, path:, title:, enabled_description:, disabled_description:, alt_text: nil, show_button: true,
```

Comment:
The name `toggle` is not into very intuitive. I was wondering for awhile, which toggle should be shown. I would've expected something like `enabled` which also matches the string keys you used. What do you think?